turn/src/server/server_test.rs

```diff
@@ -93,15 +94,15 @@ async fn test_server_simple() -> Result<()> {
     Ok(())
 }
 
-struct VNet {
-    wan: Arc<Mutex<router::Router>>,
-    net0: Arc<net::Net>,
-    net1: Arc<net::Net>,
-    netl0: Arc<net::Net>,
-    server: Server,
+pub struct VNet {
```

Comment:
Why these changes?

Reply:
It is not actual now, as the tests will be reworked.

Reply:
Aight, please revert it if it's not gonna be used

Reply:
Sure